ivy/functional/frontends/torch/reduction_ops.py

```diff
@@ -182,3 +182,8 @@ def quantile(input, q, dim=None, keepdim=False, *, interpolation="linear", out=N
 @to_ivy_arrays_and_back
 def count_nonzero(input, dim=None):
     return ivy.count_nonzero(input, axis=dim).astype(ivy.int64)
+
+
+@to_ivy_arrays_and_back
+def mode(input, dim, keepdim=False, *, out=None):
+    return ivy.mode(input, axis=dim, keepdims=keepdim, out=out)

```

Comment:
I don't think we've got a function named `mode` in our api. This is causing the tests to fail due to unresolved reference.

Reply:
Then how to implement it.

Reply:
Or attached it with Tensor.mode function

Reply:
Mode simply finds the most common value in each row of the input tensor. This is something you can implement using a composition of existing ivy functions.

Reply:
Ok understand thanks